config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml

```diff
@@ -0,0 +1,168 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
```

Comment:
I'm not 100% sure we can have two sample with the same resource name.

Reply:
oh good catch, missed that

Reply:
done